app/src/main/java/ch/epfl/cs311/wanderwave/di/ServiceModule.kt

```diff
@@ -46,6 +46,7 @@ object ServiceModule {
   @Singleton
   fun provideAppDatabase(@ApplicationContext context: Context): AppDatabase {
     return Room.databaseBuilder(context.applicationContext, AppDatabase::class.java, "app_database")
+        .fallbackToDestructiveMigration()
```

Comment:
Why is this in here, I don't see any changes to the local DB. Is it just to avoid crashing if someone has a very old version installed?

Reply:
I had a very old database apparently and it was making the CI and the app crash I don't know why, not it's working